plugins/woocommerce/tests/legacy/unit-tests/order/class-wc-tests-crud-orders.php

```diff
@@ -611,6 +611,190 @@ public function test_calculate_shipping() {
 		$this->assertEquals( 20, $object->get_shipping_total() );
 	}
 
+	public function test_calculate_shipping_with_shipping_classes() {
```

Comment:
Thanks for adding this! However, we should probably move it to a new location. As [noted here](https://github.com/woocommerce/woocommerce/blob/7.7.2/plugins/woocommerce/tests/README.md#writing-tests):

> - `tests/legacy/unit-tests` contains tests for code in the includes directory. No new tests should be added here, ever; existing test classes shouldn't get new tests either. Fixing faulty existing tests is allowed.
> - `tests/php/includes` is where all the new tests for code in the includes directory should be written.

In this case we are testing code in the includes directory, so the test belongs in `tests/php/includes`. I suggest a new path:

`tests/php/includes/shipping/flat-rate/class-wc-shipping-flat-rate-test.php`

Reply:
Sounds good @barryhughes! Created the file and transferred the code over there!